uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1394,19 +1394,135 @@ sub make_nii {
     # Get MINC filename and NIfTI filename
     my $file = $$fileref;
     my $minc  = $file->getFileDatum('File');
-    my $nifti = $minc;
-    $nifti    =~ s/mnc$/nii/g;
+    my ($nifti, $bval_file, $bvec_file) = map { $minc }(0 .. 2);
+    $nifti     =~ s/mnc$/nii/g;
+    $bval_file =~ s/mnc$/bval/g;
+    $bvec_file =~ s/mnc$/bvec/g;
 
     #  mnc2nii command
     my $m2n_cmd = "mnc2nii -nii -quiet " .
                     $data_dir . "/" . $minc . " " .
                     $data_dir . "/" . $nifti;
     system($m2n_cmd);
 
+    # create complementary nifti files for DWI acquisitions
```

Comment:
Again, I think `"$data_dir/$bval_file"` and `"$data_dir/$bvec_file"` are clearer.